charts/wire-server-metrics/values.yaml

```diff
@@ -52,7 +52,7 @@ prometheus-operator:
             options:
               path: /var/lib/grafana/dashboards/default
 
-    # TODO: Chart importing is broken for the time being; should be fixed in 
+    # TODO: Chart importing is broken for the time being; should be fixed in
```

Comment:
PR https://github.com/helm/charts/pull/12512 is now merged, has this issue been resolved? Can grafana dashboards now be imported? Please update this comment.

Reply:
> PR helm/charts#12512 is now merged, has this issue been resolved? Can grafana dashboards now be imported? Please update this comment.

It should be, but I think it might require a more recent helm version. In either case, I'd like to do that separately since I also want to update the dashbords

Reply:
Actually, ignore my previous comment, I will try to see if this got sorted out.

Reply:
Indeed it got fixed with: https://github.com/wireapp/wire-server-deploy/pull/133/commits/e36874211f29f5de495a05ed3e338c867da8b35b